### DISCUSSION TITLE
How to support the project besides writing code?

### DISCUSSION THREAD

**Discussion by msvana** [flxzt/rnote#162] [General]
Hello, I really like what you are doing here with rnote, I was looking for a similar note-taking app for a long time. I want to support the project in some way, but I am not a RUST developer myself. So I was wondering: What can I do to support the project besides writing code?

EDIT: Just noticed the Donete buttons ...

**Comment by flxzt** [comment]
Hi, thank you for the donation! If you want you could start and maintain a czech translation, more translations are always very welcome :)
